resource-managers/kubernetes/integration-tests/src/test/resources/log4j.properties

```diff
@@ -15,8 +15,14 @@
 # limitations under the License.
 #
 
-# Set everything to be logged to the file target/integration-tests.log
-log4j.rootCategory=INFO, file
+# Set everything to be logged to the file target/integration-tests.log and the console
+log4j.rootCategory=INFO, file, stdout
```

Comment:
I am not sure we want console here. On the upside, I understand most people actually do not know they can check target/integration-tests.log. On the downside, too many log messages could get people lost.

Anyone has a strong opinion on this?

Reply:
Ping! @echarles or anyone else.

Reply:
At this point, I am good with this PR pending this question being resolved. Maybe we just want to remove console output part and move on so the PR can be merged?